src/jdk.compiler/share/classes/com/sun/tools/javac/code/Source.java

```diff
@@ -217,7 +223,7 @@ public Target requiredTarget() {
         PATTERN_MATCHING_IN_INSTANCEOF(JDK16, Fragments.FeaturePatternMatchingInstanceof, DiagKind.NORMAL),
         REIFIABLE_TYPES_INSTANCEOF(JDK16, Fragments.FeatureReifiableTypesInstanceof, DiagKind.PLURAL),
         RECORDS(JDK16, Fragments.FeatureRecords, DiagKind.PLURAL),
-        SEALED_CLASSES(JDK16, Fragments.FeatureSealedClasses, DiagKind.PLURAL),
+        SEALED_CLASSES(JDK17, Fragments.FeatureSealedClasses, DiagKind.PLURAL),
```

Comment:
Not sure if this is right - is it changed because it is still preview?

Reply:
Right; JEP 397: "Sealed Classes (Second Preview)" is PTT for JDk 16.